### ISSUE TITLE
Investigate adapted adaptive behavior

### ISSUE THREAD

**Issue by Willem3141** [graphchecker/moodle-qtype_graphchecker#31]
In GitLab by @Willem3141 on Feb 24, 2020, 18:04

Annoyingly, the `qbehaviour_adaptive_adapted_for_coderunner` behavior [actively prevents non-CodeRunner questions from using it](https://github.com/trampgeek/moodle-qbehaviour_adaptive_adapted_for_coderunner/blob/master/behaviour.php#L51).

We should either fork that behavior just to remove / change that check (ugly!), or figure out if there is a way to run with the default behavior.

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Mar 4, 2020, 12:14

For now we can just have a patch. Jan Willem was able to install that on the test Moodle server.

 *By Willem Sonke on 2020-03-04T11:14:02 (imported from GitLab project)*

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Feb 25, 2020, 11:23

After discussion with Arthur:

* we'll first do the ugly solution (fork the behavior)
* we also asked Richard (CodeRunner author) if he can fix it upstream, or he has a different solution

 *By Willem Sonke on 2020-02-25T10:23:45 (imported from GitLab project)*

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on May 22, 2020, 12:24

unassigned @Willem3141

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Jun 2, 2020, 01:24

removed milestone

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Jun 2, 2020, 01:24

Keeping this open to track caching for the deferred behavior.

However, after discussion with Arthur, we'll move this to after the exam week for now.

**Comment by Willem3141** [comment]
In GitLab by @Willem3141 on Jun 12, 2020, 21:28

There was an issue with deferred behavior not showing the result tables, not even when . Therefore Arthur and I decided to tackle this issue now. After a two-hour video call, we figured out how the adapted adaptive behavior works exactly, and made a new question behavior ‘adapted deferred feedback’ (d2a7cc79647e688b5a3f3859360c1a3444f52039 and repository https://gitlab.tue.nl/wsonke/moodle-qbehavior_deferredfeedback_graphchecker). So now adaptive and deferred both work nicely with GraphChecker.
